eng/targets/Helix.Common.props

```diff
@@ -21,7 +21,7 @@
   <!-- PR(ci.yaml) required queues -->
   <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64'">
     <HelixAvailableTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Windows.10.Amd64.Open" Platform="Windows" />
+    <HelixAvailableTargetQueue Include="Windows.10.Amd64.Server20H1.Open" Platform="Windows" />
```

Comment:
How many machines are in this target queue? I'd be concerned if there isn't enough machines.

Reply:
We've already been using it for nightly runs. The more we use it the more machines they allocate to it. There may be a temporary lag while it catches up.